resources/js/pages/admin/emojis/index.tsx

```
@@ -0,0 +1,161 @@
+
+                {/* Pagination */}
+                {emojis.links && emojis.links.length > 3 && (
+                    <div className="flex items-center justify-center space-x-2">
+                        {emojis.links.map((link, index) => (
+                            <Button
+                                key={index}
+                                variant={link.active ? 'default' : 'outline'}
+                                size="sm"
+                                disabled={!link.url}
+                                onClick={() => link.url && router.get(link.url)}
+                                dangerouslySetInnerHTML={{ __html: link.label }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)